ConsumerApi.Tests.Integration/Features/Devices/{id}/DELETE.feature

```diff
@@ -3,33 +3,31 @@ Feature: DELETE Device
 
 User deletes an un-onboarded Device
 
-Scenario: Deleting an un-onboarded Device
-	Given an Identity i with a device d1
-	And an un-onboarded device d2
-	When a DELETE request is sent to the Devices/{id} endpoint with d2.Id
-	Then the response status code is 204 (Ok)
-	And d2 is deleted
+	Scenario: Deleting an un-onboarded Device
+		Given an Identity i with a device d1
+		And an un-onboarded device d2 that belongs to i
+		When i sends a DELETE request to the /Devices/{id} endpoint with d2.Id
+		Then the response status code is 204 (Ok)
+		And d2 of i is deleted
```

Comment:
But the devices are retrieved based on the identity to which they belong:
```csharp
    protected async Task<ApiResponse<ListDevicesResponse>> ListDevices(string identityName = "i")
    {
        var response = await Identity(identityName).Devices.ListDevices();
        response.Should().BeASuccess();

        return response;
    }
```

Since `i` is the default identity name I could omit in this and similar cases, but I prefer to explicitly state it for clarity. I don't know how you feel about it.

Reply:
Changes made.